lib/msal-browser/src/crypto/SignedHttpRequest.ts

```diff
@@ -24,12 +25,16 @@ export class SignedHttpRequest {
 
     constructor(
         shrParameters: SignedHttpRequestParameters,
+        performanceClient: IPerformanceClient,
```

Comment:
We should avoid this here as this class is publicly used but we don't expect customers to implement their own PerformanceClient